action/protocol/execution/protocol_test.go

```diff
@@ -3,7 +3,7 @@
 // or fitness for purpose and, to the extent permitted by law, all liability for your use of the code is disclaimed.
 // This source code is governed by Apache License 2.0 that can be found in the LICENSE file.
 
-package execution
+package execution_test
```

Comment:
why need to change this

Reply:
to solve import cycle issue between execution and staking